packages/react/src/components/modal/styled.ts

```diff
@@ -0,0 +1,97 @@
+import { CSSProperties } from 'react';
+import ReactModal from 'react-modal';
+import styled from 'styled-components';
+import { IconButton } from '../buttons/icon-button';
+import { ContentProps, FooterProps, HeaderProps, StyledModalProps } from './types';
+
+function getModalMinWidth({ breakpoints, isMobile }: StyledModalProps): string {
+    return isMobile ? 'initial' : `calc(${breakpoints.mobile}px - var(--spacing-4x))`;
+}
+
+function getModalWidth({ $width, isMobile }: StyledModalProps): CSSProperties['width'] {
+    return isMobile ? 'calc(100vw - var(--spacing-2x))' : $width;
+}
+
+function getWidthPadding({ $noPadding, isMobile }: ContentProps): string {
+    if ($noPadding) {
+        return '0';
+    }
+    if (isMobile) {
+        return 'var(--spacing-2x)';
+    }
+    return 'var(--spacing-4x)';
+}
+
+function getHeightPadding({ $noPadding, isMobile }: ContentProps): string {
+    if ($noPadding) {
+        return '0';
+    }
+    if (isMobile) {
+        return 'var(--spacing-2x)';
+    }
+    return 'var(--spacing-3x)';
+}
```

Comment:
[MAY] Qu'est-ce que tu pense de ça ?
Est-ce que tu trouves que c'est toujours aussi explicite que deux if ? 🤔 
Même chose plus haut.
```suggestion
function getHeightPadding({ $noPadding, isMobile }: ContentProps): string {
    return $noPadding ? '0' : isMobile ? 'var(--spacing-2x)' : 'var(--spacing-3x)';
}
```

Reply:
Nos règles ESLint nous empêche de faire des nested _ternary expressions_!